src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Figure [] figures = new Figure[] {new Square(new Random().nextDouble()),
+                new Rectangle(new Random().nextDouble(), new Random().nextDouble()),
+                new RightTriangle(new Random().nextDouble(), new Random().nextDouble()),
+                new Circle(new Random().nextDouble()),
+                new IsoscelesTrapezoid(new Random().nextDouble(),
+                        new Random().nextDouble(), new Random().nextDouble())};
+        int index = new Random().nextInt(figures.length);
+        return figures[index];
+    }
```

Comment:
we are creating a lot of instances of `Random`
let's create field in the class and reuse it

also we want to return only one figure, so we don't want to create unnecessary instances (maybe switch case will help)